misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -192,22 +230,17 @@ def annotate_logged_errors(log_files: list[str]) -> int:
     # Keep track of known errors so we log each only once
     already_reported_issue_numbers: set[int] = set()
 
-    for error in error_logs:
-        error_message = sanitize_text(error.match.decode("utf-8"))
-        formatted_error_message = f"```\n{error_message}\n```"
-
-        for artifact in artifacts:
-            if artifact["job_id"] == job and artifact["path"] == error.file:
-                org = os.environ["BUILDKITE_ORGANIZATION_SLUG"]
-                pipeline = os.environ["BUILDKITE_PIPELINE_SLUG"]
-                build = os.environ["BUILDKITE_BUILD_NUMBER"]
-                linked_file = f'[{error.file}](https://buildkite.com/organizations/{org}/pipelines/{pipeline}/builds/{build}/jobs/{artifact["job_id"]}/artifacts/{artifact["id"]})'
-                break
-        else:
-            linked_file = error.file
+    def handle_error(error_message: bytes, location: str):
+        # Don't have too huge output
+        print_error_message = (
```

Comment:
Do we need this in other places as well? Consider moving it into `buildkite.py` as utility.